examples/layer-browser/src/app.js

```diff
@@ -111,6 +111,12 @@ class App extends PureComponent {
     this.setState({clickedItem: info});
   }
 
+  _onQueryObjects() {
+    const {width, height} = this.state;
+    const infos = this.refs.deckgl.queryObjects({x: 0, y: 0, width, height});
+    console.log(infos); // eslint-disable-line
```

Comment:
If the console is not open, clicking the "Query Rendered Objects" button will appear to do nothing. Probably better to pop open a div with `JSON.stringify(infos)` in it.